release notes/v0.51.0.md

```diff
@@ -6,10 +6,22 @@ k6 `v0.51.0` is here 🎉! This release includes:
 
 ## Breaking changes
 
-- `#pr`, `<small_break_1>`
-- `#pr`, `<small_break_2>`
+### Browser APIs to Async
 
-### (_optional h3_) `<big_breaking_change>` `#pr`
+In the last release notes we mentioned this breaking change, and we wanted to remind and update you on the plan. In the **next** release, most of the synchronous browser APIs will be migrated to be asynchronous (promisifying them). We expect this will affect most if not all of our users.
```

Comment:
```suggestion
In the last release notes [we mentioned](ADD HERE A LINK) this breaking change, and we wanted to remind and update you on the plan. In the **next** release (v0.52.0), most of the synchronous browser APIs will be migrated to be asynchronous (promisifying them). We expect this will affect most if not all of our users.
```
We may link an issue so users can consider subscribing to it.